src/main/java/org/team340/robot/Constants.java

```diff
@@ -35,5 +37,10 @@ public static final class RobotMap {
         public static final int kBrEncoder = 13;
 
         public static final int kCanandgyro = 14;
+
+        public static final int kGooseneckRollersMotor = 30;
+        public static final int kGooseneckRollersBeamBreak = 31;
```

Comment:
```suggestion
        public static final int kGooseneckRollersMotor = 31;
        public static final int kGooseneckCANdi = 32;
```
Forgot to mention this at last night's meeting, the CANdi on the gooseneck is also used for the absolute encoder on the pivot, so the name should be generalized. Also, we should bump the CAN IDs so that the pivot motor can be 30.